tests/lib/rook/upstream.py

```diff
@@ -33,12 +33,6 @@ def __init__(self, workspace, kubernetes):
         self.ceph_dir = os.path.join(
             self.build_dir, 'cluster/examples/kubernetes/ceph')
 
-    def preinstall(self):
```

Comment:
Why this move of the function? Makes reviews harder that it could be

Reply:
That's because I discovered this bug when testing helm, and cherry picked from other branch where other change were done.
I agree this is not optimal but I wanted to split this from helm PR.

Reply:
Can you undo the move here then? It's useless to merge those changes imo

Reply:
Done